scripts/install_dependencies.sh

```diff
@@ -17,6 +17,6 @@ pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${CC_PLUGIN_IMOS_URL} \
     | python -c "import json, sys; print json.load(sys.stdin)['${CC_PLUGIN_IMOS_URL}']['local_file']")
 pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${AODNCORE_URL} \
     | python -c "import json, sys; print json.load(sys.stdin)['${AODNCORE_URL}']['local_file']")
-pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${NCWRITER_URL} \
-    | python -c "import json, sys; print json.load(sys.stdin)['${NCWRITER_URL}']['local_file']")
+pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${AODNTOOLS_URL} \
+    | python -c "import json, sys; print json.load(sys.stdin)['${AODNTOOLS_URL}']['local_file']")
 pip install -r requirements.txt
```

Comment:
Could add a `pip install httpretty==0.9.6` at the end, just so all the tests pass on a fresh install.

Reply:
Ah yeah, thought I'd fixed that before but didn't. I've added the optional testing requirements in the "standard" way and added to the script.